scripts.html

```diff
@@ -4,16 +4,219 @@
 menu-id: l1-workflow2
 pdfs:
     - scripts.pdf
-description: Scripts allow users to add server-side functionality to the OMERO clients.
+description: Scripts allow additional server-side functionality to be added by users to the OMERO clients.
 version:
     - yes
```

Comment:
@gusferguson: the `redirect-from` is the way we mitigated the SLS renaming impact i.e. it redirects http://help.openmicroscopy.org/cls-public-demo.html to http://help.openmicroscopy.org/sls-public-demo.html - see https://github.com/openmicroscopy/ome-help/pull/119.